src/com/raghav/javabasics/day4/FindFirstNonRepeatingCharAscii.java

```diff
@@ -0,0 +1,42 @@
+package com.raghav.javabasics.day4;
+//Given a string, write a program to print the first non-repeating character in the String.
+
+public class FindFirstNonRepeatingCharAscii {
+    public static void main(String args[]) {
+        FindFirstNonRepeatingCharAscii result = new FindFirstNonRepeatingCharAscii();
+        result.findNonRepeatingChar("abc");
```

Comment:
Output: 
a
c
No non repetitive num in string
c
h
Array is Empty/Null
Array is Empty/Null